specifyweb/frontend/js_src/lib/components/UserPreferences/index.tsx

```diff
@@ -148,11 +148,14 @@ export function PreferencesContent({
               {description !== undefined && <p>{description}</p>}
               {subCategories.map(
                 ([subcategory, { title, description = undefined, items }]) => (
-                  <section className="flex flex-col gap-4" key={subcategory}>
-                    <div className="flex items-center">
-                      <span className="flex-1" />
+                  <section
+                    className="flex flex-col items-start gap-4 md:items-stretch"
+                    key={subcategory}
+                  >
+                    <div className="flex items-center gap-2">
+                      <span className="hidden md:flex-1" />
```

Comment:
With `hidden`, this element is never displayed. Either remove `hidden` or remove the element